node/flags/string_to_uint64.go

```diff
@@ -30,7 +30,7 @@ func (s *StringToUint64Value) Set(val string) error {
 			return err
 		}
 		key := strings.TrimSpace(parts[0])
-		s.value[key] = out
+		(*s.value)[key] = out
```

Comment:
Why is `Set` not overwriting the map? It adds the parsed string to existing values which is different than `Set` should behave?

Also there is no check against a possible `nil` pointer exception here?

Reply:
>Why is Set not overwriting the map? 

To support the use case of passing the flag multiple times like `-a foo=100 -a bar=300`. There is even a test for it somewhere.

>It adds the parsed string to existing values which is different than Set should behave?

The `Value` interface doesn't specify how `Set()` should behave. There are even implementations in `pflag` library that behave like this (appending), for example [StringSliceVar](https://pkg.go.dev/github.com/spf13/pflag@v1.0.5#FlagSet.StringSliceVar).

>Also there is no check against a possible nil pointer exception here?

Why would there be a nil check here? It's not valid to create an instance of `StringToUint64Value` with a `nil` value in the first place.